wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -485,27 +522,53 @@ private data class LocationAndComments(val comment: String, val loc: Location)
 private class SourceCodeHelper(
   fd: FileDescriptorProto
 ) {
-  val locs: Map<List<Int>, SourceCodeInfo.Location> = makeLocationMap(fd.sourceCodeInfo.locationList)
+  val locs: Map<List<Int>, List<SourceCodeInfo.Location>> = makeLocationMap(fd.sourceCodeInfo.locationList)
```

Comment:
The changes from here to the end might be a bit confusing. But it's due to the way that extend _blocks_ are not explicitly modeled in the descriptor, so the source code info is a bit weird.

You can read more about it here:
https://www.protobuf.com/docs/descriptors#source-code-info-for-extension-blocks